test/testdata/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.24.4
 
 require (
 	github.com/SanteonNL/go-fhir-client v0.5.0
+	github.com/nuts-foundation/nuts-knooppunt v0.1.1
```

Comment:
we don't want this, because;
- it's a circular dependency now
- it makes testing slow, since the knooppunt-tests use this testdata package, which then goes on to download some (outdated) version of nuts-knooppunt

If we like the util funcs you added, you can add them to the `zorgbijjou/golang-fhir-models` library in the `caramel` package (because if you put syntactic sugar on FHIR/FIRE, you get caramel, you know...)